client/src/pages/Wishlist/WishList.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+import { useCart } from '../../contexts/CartContext';
+import { useWishlist } from '../../contexts/WishContext';
+import Spinner from "../../utils/Spinner";
+import Toast from "../../utils/Toast";
+
+const WishList = () => {
+
+  useEffect(() => {
+    window.scroll({ behavior:'smooth',top:0 });
+  },[])
+
+  const {wishlist,handleWishlist} = useWishlist();
+  const {cart, addToCart } = useCart();
+  const [spinner,setSpinner] = useState(false);
+  const [error,setError] = useState(false);
+
+  return (
+      <div className="wishlist-container">
+          <Toast show={error} onClick={() => setError(false)} message="Something went wrong with server" error={true} background="red" color="white"/>
+
+      <Spinner show={spinner}/>
+        <h1 style={{textAlign: 'center'}}>Wishlist ({wishlist.length})</h1>
+        <div style={{display: wishlist.length > 0 ? 'grid': "block"}}  className="products">
+              {
+                 wishlist.length > 0 ? wishlist.map( item => (
+                    <div key={item._id} className={ item.stock ? "card" : "card out-of-stock"}>
+                      <div className="remove-icon">
```

Comment:
fix remove icon. seee this
https://shareit.video/ItBWfU6AxOv1yzTNCNgo

Reply:
fixed